### PR TITLE
@broskoski: Mobile breakpoints for Artsy Primer personalize flow

### PR DESCRIPTION
**Pull request by craigspaeth**
This adds mobile responsive breakpoints to the Artsy Primer personalize flow to get a quick mobile friendly version working. These aren't 100% to David's specs b/c some of the UI is trickier to morph into what he designed, so I made some off-the-cuff compromises to get something nice looking out the door sooner. I'll follow up in an email with more details about handing off this project to Collector Experience Team.

cap

### CODE REVIEW

File: desktop/apps/artsy_primer/personalize/stylesheets/favorites.styl [hunk 1]
```diff
@@ -3,6 +3,15 @@
   max-width 1080px
   &[data-step='favorites']
     width 80%
+    @media screen and (max-width: 768px)
```
Comment (broskoski) [inline desktop/apps/artsy_primer/personalize/stylesheets/favorites.styl]: Maybe we should make a stylus mixin for these, `768px` could be something easily forgettable.

File: desktop/apps/auth/index.styl [hunk 2]
```diff
@@ -3,7 +3,7 @@
 #auth-page
 #reset-password-page
   text-align center
-  width 400px
+  max-width 400px
```
Comment (broskoski) [inline desktop/apps/auth/index.styl]: 👍

**Comment by broskoski** [comment]
LGTM! Feel free to send any additional work my way.

**Comment by joeyAghion** [comment]
Vapor failed to find the "skip" button in the personalization flow overnight. It appears to be a CSS regression that pushes the button beyond the left margin (whereas it used to be just to the right of the header):



I am guessing this PR is responsible but am not 100% sure. Care to redirect me if not?

**Comment by mzikherman** [comment]
Sorry I think i introduced this actually!

Fixed in https://github.com/artsy/force/pull/1022

**Pull request merged**